packages/kit/src/exports/vite/index.js

```diff
@@ -797,7 +797,7 @@ Tips:
 				const log = logger({ verbose });
 
 				/** @type {import('vite').Manifest} */
-				const server_manifest = JSON.parse(read(`${out}/server/${vite_config.build.manifest}`));
+				const server_manifest = JSON.parse(read(`${out}/server/.vite/manifest.json`));
```

Comment:
I think it's probably still better to use the configured value in case the user changes it (not sure if this is one we allow them to change or not, but still seems slightly preferable in case we wanted to loosen that restriction as we gradually have loosened others over time)
```suggestion
				const server_manifest = JSON.parse(read(`${out}/server/${vite_config.build.manifest}`));
```

Reply:
We should add a boolean check here. `true` and `false` are possible values which doesn’t work well with the string template (`true` will cause Vite to use the default location but the path is not stored in the config setting)

Reply:
yeah, just noticed that's why you did that as the tests were failing

Reply:
Thanks for checking!